satflow/models/hub.py

```diff
@@ -27,6 +27,50 @@
     hf_hub_url = None
     cached_download = None
 
+from huggingface_hub import ModelHubMixin, PYTORCH_WEIGHTS_NAME, CONFIG_NAME, hf_hub_download
```

Comment:
I'm not entirely sure what the reasoning is behind the try / except blocks for the `torch.hub` and `huggingface_hub` imports - should I include my mixin imports in there as well?

Reply:
The try/except is kinda just left from when I copied it from `timm`'s repo, so no need to include your imports there either.